client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -214,7 +214,17 @@ public class RssSparkConfig {
   public static final ConfigEntry<Integer> RSS_CLIENT_ASSIGNMENT_RETRY_TIMES = createIntegerBuilder(
           new ConfigBuilder(SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_ASSIGNMENT_RETRY_TIMES))
           .createWithDefault(RssClientConfig.RSS_CLIENT_ASSIGNMENT_RETRY_TIMES_DEFAULT_VALUE);
-  
+
+  public static final ConfigEntry<Long> RSS_CLIENT_FALLBACK_RETRY_INTERVAL = createLongBuilder(
```

Comment:
Could we use`spark.rss.client.access.retry.interval.ms`? Because we want to access RSS instead of fallback, the variable's name contains unit, it will improve readability.

Reply:
done.

Reply:
Could we change `RSS_CLIENT_FALLBACK_RETRY_INTERVAL`  to `RSS_CLIENT_ACCESS_RETRY_INTERVAL_MS` together?